classes/sushi/IR.php

```diff
@@ -243,11 +245,16 @@ public function getReportItems(): array
                 'Data' => __('sushi.exception.3030', ['beginDate' => $this->beginDate, 'endDate' => $this->endDate])
             ]);
         }
+        return $results;
+    }
+
+    /** Get report items */
+    public function getReportItems(): array
+    {
+        $results = $this->getQueryResults();
 
         $resultsGroupedBySubmission = $items = [];
```

Comment:
`$resultsGroupedBySubmission` is immediately overwritten, so this line can be removed.

Reply:
I also just noticed this means `$items is being set to an empty array as well. I think only `$reusltsGroupedBySubmisison` needs to be removed here.